### PR TITLE
Medieval Mod Series 2

### PR DESCRIPTION
**Pull request by chaosvolt**
Armor, delicious armor. The only armor I'm holding back (for now) would be the shields, as I'm not yet sure if they're implementable as-is.

Armor tweaks:
1. Boosted plate armor's protection to 18/18, comparable to the equally-encumbering plated leather armor and the less-encumbering chainmail.
2. Repurposed metal leg guards into proper greaves with a leather lining, while also reducing their protection to 18/18. This keeps them in line with the bronze greaves introduced.

Armor additions:
1. Corinthian helm.
2. Galea, or the classic type of imperial Roman helm.
3. Nasal helm, for proper vikings.
4. Conical helm, wish I knew the correct Mongolian word for the major types of helm used by the Mongol Empire.
5. Tunic. More professional than a rag tunic, and has sleeves.
6. Gambeson, because mail with padding is a Bad Idea.
7. Bell cuirass, the basic bronze breastplate.
8. Lorica Segmentata, another fine option for metal armor.
9. An option to bundle the chainmail vest, sleeves, and leggings as a hauberk, similar to how a few other "suit made from conjoined pieces" have alternatives that don't use the full ensemble.
10. A type of light cotton (linen would be more accurate) kilt that will be relevant once I implement the professions from the full version of the mod.
11. Bronze greaves.
12. Lamallar armor, this version made from leather. For comparison, the lorica segmentata is considered a type of laminar armor.

### CODE REVIEW

**Comment by Rivet-the-Zombie** [comment]
How about an update that starts pulling this stuff into the main game?

**Comment by chaosvolt** [comment]
Doh. Would be fine by me, but how much of the content?

The buff to plate armor and tweak to metal leg guards might be on my to-to list for one, but unsure about all the other content.

**Comment by Coolthulhu** [comment]
Buffs and possibly some of that cloth armor, provided it is balanced against core items (didn't check yet).

**Comment by chaosvolt** [comment]
Most of the cloth stuff is clothing, the tunic and shendyt.

The gambeson should be okay, not sure of its encumbrance versus protection, but it does have the potentially useful shtick of being padding for underneath clothing.

**Comment by chaosvolt** [comment]
Hmm. Should I include mainlining of stuff in this PR, or start a separate one?

**Comment by Coolthulhu** [comment]
Balancing can be here, but mainlining should be separate

**Comment by chaosvolt** [comment]
Ah, okay. Another reason why I'm generally in favor of adding as mod content first, then determining what to mainline. ^^"

File: data/mods/Medieval_Stuff/armor.json [hunk 1]
```diff
@@ -0,0 +1,294 @@
...
+        "symbol" : "[",
+        "description" : "A simple garment long enough to cover down to the knees.",
+        "price" : 4200,
+        "material" : ["cotton", "null"],
+        "volume" : 5,
+        "cutting" : 0,
+        "warmth" : 2,
+        "phase" : "solid",
+        "environmental_protection" : 0,
+        "encumbrance" : 0,
```
Comment (Coolthulhu) [inline data/mods/Medieval_Stuff/armor.json]: Sleeveless tunic is 3 encumbrance

**Comment by chaosvolt** [comment]
Hope it goes well.

**Comment by chaosvolt** [comment]
Ack. It didn't show the notes until I commented. >.<

File: data/mods/Medieval_Stuff/armor.json [hunk 2]
```diff
@@ -0,0 +1,294 @@
...
+        "volume" : 8,
+        "cutting" : 0,
+        "warmth" : 30,
+        "phase" : "solid",
+        "environmental_protection" : 0,
+        "encumbrance" : 10,
+        "bashing" : -2,
+        "flags" : ["VARSIZE", "SKINTIGHT", "STURDY"],
+        "coverage" : 80,
+        "material_thickness" : 5
```
Comment (Coolthulhu) [inline data/mods/Medieval_Stuff/armor.json]: That's 8/8 protection at 5 encumbrance (when fit), skintight. That 80% coverage isn't enough to make it anything but too good. OK in a mod, but unacceptably good in mainline.

File: data/mods/Medieval_Stuff/armor.json [hunk 3]
```diff
@@ -0,0 +1,294 @@
...
+        "weight" : 453,
+        "color" : "light_gray",
+        "covers" : ["TORSO", "ARMS", "LEGS"],
+        "to_hit" : 0,
+        "storage" : 0,
+        "symbol" : "[",
+        "description" : "A thick jacket of quited fabric, designed to be worn underneath mail or other armor.  Or worn on its own, if you can't afford proper armor.",
+        "price" : 20000,
+        "material" : ["cotton", "null"],
+        "volume" : 8,
```
Comment (Coolthulhu) [inline data/mods/Medieval_Stuff/armor.json]: That's a very low volume

File: data/mods/Medieval_Stuff/armor.json [hunk 4]
```diff
@@ -0,0 +1,294 @@
...
+        "type" : "ARMOR",
+        "id" : "gambeson",
+        "name" : "gambeson",
+        "weight" : 453,
+        "color" : "light_gray",
+        "covers" : ["TORSO", "ARMS", "LEGS"],
+        "to_hit" : 0,
+        "storage" : 0,
+        "symbol" : "[",
+        "description" : "A thick jacket of quited fabric, designed to be worn underneath mail or other armor.  Or worn on its own, if you can't afford proper armor.",
```
Comment (Coolthulhu) [inline data/mods/Medieval_Stuff/armor.json]: Shouldn't it be "quilted"?

**Comment by chaosvolt** [comment]
Hmm. So...bump up tunic's encumbrance to match the sleeveless version, bump up gambeson's volume and encumbrance...to how much?

File: data/mods/Medieval_Stuff/armor.json [hunk 5]
```diff
@@ -0,0 +1,294 @@
...
+        "volume" : 24,
+        "cutting" : 0,
+        "warmth" : 15,
+        "phase" : "solid",
+        "environmental_protection" : 0,
+        "encumbrance" : 15,
+        "bashing" : 6,
+        "flags" : ["VARSIZE", "OUTER", "STURDY"],
+        "coverage" : 90,
+        "material_thickness" : 3
```
Comment (Coolthulhu) [inline data/mods/Medieval_Stuff/armor.json]: Not really OP, but it's comparable to modern kevlar MBR.

File: data/mods/Medieval_Stuff/armor.json [hunk 6]
```diff
@@ -0,0 +1,294 @@
...
+        "volume" : 24,
+        "cutting" : 0,
+        "warmth" : 15,
+        "phase" : "solid",
+        "environmental_protection" : 0,
+        "encumbrance" : 15,
+        "bashing" : 6,
+        "flags" : ["VARSIZE", "OUTER", "STURDY"],
+        "coverage" : 90,
+        "material_thickness" : 4
```
Comment (Coolthulhu) [inline data/mods/Medieval_Stuff/armor.json]: Same armor value as plate armor, with less than half the encumbrance. Has lower coverage, but still.

**Comment by chaosvolt** [comment]
Hmm. That is a bit for bronze, so I could bump that down to somewhere between the 18 of most of the medieval armors and its current level.

Guess I'll work on those real quick?

File: data/mods/Medieval_Stuff/armor.json [hunk 7]
```diff
@@ -0,0 +1,294 @@
...
+        "volume" : 10,
+        "cutting" : 0,
+        "warmth" : 0,
+        "phase" : "solid",
+        "environmental_protection" : 0,
+        "encumbrance" : 15,
+        "bashing" : -3,
+        "flags" : ["VARSIZE", "STURDY"],
+        "coverage" : 95,
+        "material_thickness" : 3
```
Comment (Coolthulhu) [inline data/mods/Medieval_Stuff/armor.json]: As above, except slightly better because coverage.

**Comment by chaosvolt** [comment]
> As above, except slightly better because coverage.

That should be taken straight from the stats for the vanilla mail armor. >_>

**Comment by Coolthulhu** [comment]
Vanilla iron armors are kinda crap for some reason.

**Comment by chaosvolt** [comment]
Odd. Just checked and yeah, regular mail is all 18/18. Should that be nerfed?

File: data/mods/Medieval_Stuff/armor.json [hunk 8]
```diff
@@ -0,0 +1,294 @@
...
+        "material" : ["bronze", "leather"],
+        "volume" : 22,
+        "cutting" : 0,
+        "warmth" : 15,
+        "phase" : "solid",
+        "environmental_protection" : 1,
+        "encumbrance" : 20,
+        "bashing" : -5,
+        "coverage" : 70,
+        "material_thickness" : 4
```
Comment (Coolthulhu) [inline data/mods/Medieval_Stuff/armor.json]: Better protection than iron ones.

File: data/mods/Medieval_Stuff/armor.json [hunk 9]
```diff
@@ -0,0 +1,294 @@
...
+        "symbol" : "[",
+        "description" : "A pair of bronze leg guards with a simple leather lining.",
+        "flags" : ["BELTED", "STURDY"],
+        "price" : 40000,
+        "material" : ["bronze", "leather"],
+        "volume" : 22,
+        "cutting" : 0,
+        "warmth" : 15,
+        "phase" : "solid",
+        "environmental_protection" : 1,
```
Comment (Coolthulhu) [inline data/mods/Medieval_Stuff/armor.json]: Environmental protection on something that gives 70% coverage and is made only to resist kinetic impact?

**Comment by chaosvolt** [comment]
Better protection than...doh. Admittedly, bronze is more comparable to iron than outright superior in reality, but...hmm. Might want to make it only slightly better at most.

File: data/mods/Medieval_Stuff/recipe_armor.json [hunk 10]
```diff
@@ -0,0 +1,385 @@
...
+    [
+      [ "string_36", 4 ],
+      [ "string_6", 16 ]
+    ],
+    [
+      [ "leather", 42 ],
+      [ "tanned_hide", 7 ],
+      [ "fur", 42 ],
+      [ "tanned_pelt", 7 ]
+    ]
```
Comment (Coolthulhu) [inline data/mods/Medieval_Stuff/recipe_armor.json]: No cotton, but the armor has cotton as a material.

**Comment by chaosvolt** [comment]
Vanilla metal leg guards have 1 EP. Once again, not my screw-up. >_>

File: data/mods/Medieval_Stuff/recipe_armor.json [hunk 11]
```diff
@@ -0,0 +1,385 @@
...
+  "autolearn": true,
+  "tools": [
+    [
+      [ "needle_bone", 30 ],
+      [ "needle_wood", 30 ],
+      [ "tailors_kit", 30 ],
+      [ "sewing_kit", 30 ]
+    ]
+  ],
+  "components": [ [ [ "rag", 16 ] ] ]
```
Comment (Coolthulhu) [inline data/mods/Medieval_Stuff/recipe_armor.json]: 16 rags for something like this?
Also, shouldn't kilts be fancy? They're useless from practical point of view.

**Comment by Coolthulhu** [comment]
I just noticed that bronze has resists 5/5 while iron has 4/4. One of them is obviously wrong. I'd swap them.

**Comment by chaosvolt** [comment]
For the light kilt/shendyt, yeah. Only exists as a flavor item for a profession in the full version, who I think I could get away with giving a tunic instead (not sure of the garments).

As for bronze, it should likely be comparable to bronze. This is IRON, steel (which would be outright superior) is a separate material.

**Comment by chaosvolt** [comment]
Guessing I'll start working on messing with this, or wait for you to de-assign?

**Comment by Coolthulhu** [comment]
I'll wait a while, I'm testing the pet carrier PR atm

**Comment by chaosvolt** [comment]
Okay, will work on that.

**Comment by chaosvolt** [comment]
Hope that improves things. Since the bronze items are supposed to generally have slightly higher encumbrance and weight, it should try to imply that they're a fairly hard variant of the alloy to justify their being slightly better than the wrought iron equivalents.

Then again, most of the armor they're getting compared to, like plate armor, is steel...hmm.

**Comment by chaosvolt** [comment]
And thank you for the merge.

**Pull request merged**